### DISCUSSION TITLE
How to stop notifications about other's comments?

### DISCUSSION THREAD

**Discussion by AhmadIsmael-TomTom** [community/community#44749] [Pull Requests] [answered]
Hi, 
Imagine this scenario. Someone opens a PR and requests 5 other people and me to review it. Each of us writes a single comment, and the author replies to each one. I will get in my email a notification for each of the 5 initial comments and the next 5 replies. I am only interested in conversations that I am part of and should reply and react to, and not every comment on the PR. How can I do that?

The same thing happens with mentions. If I were mentioned in a single conversation in a PR, I would receive an email notification for every comment on that PR with `mention@noreply.github.com` as a CC. It seems that your notification system works on the PR level, and not the conversation level.

Is there a way to change that?

Thanks

**Comment by xmo-odoo** [comment, marked as answer]
AFAIK there's no way to change that, the configuration for PR notifications is incredibly lacking in flexibility.

The recommendation I have is to disable being notified by mail unless that's absolutely central to your github workflow, get your notifications in the web UI, and liberally unsubscribe (or mark as read) straight from the notifications table. It's still not great, but it's a lot better than the hell that is getting gh notifications by mail.

**Comment by Anmol-Baranwal** [comment]
Yes, there are a few ways you can customize your notifications in GitHub to receive only the relevant ones:

1. Customizing notifications for specific repositories - You can choose to receive notifications only for specific repositories you're interested in. Go to the repository you want to customize notifications for, click on the "Watch" or "Unwatch" button, and then choose the type of notifications you want to receive.

2. Customizing notifications for specific threads - If you're only interested in certain conversations within a PR, you can customize your notifications for specific threads. In the notification email, click on the "View thread" button to go to the thread, and then click on the "Unsubscribe" button at the bottom of the thread to stop receiving notifications for it.

3. Using filters in your email client - If you're receiving too many notifications, you can create filters in your email client to automatically delete or archive notifications that don't meet certain criteria (e.g., notifications that don't mention you).

Regarding your concern with mentions, unfortunately, there is no way to customize notifications at the conversation level, but you can still use the above options to customize notifications for specific repositories or threads.

> I would suggest you to give feedback on the same to GitHub using appropriate tag.

**Reply by pzuschlag** [reply]
Thaks for replying with AI content 👍🏼
really valueable 💯

**Reply by ghostinhershell** [reply]
Hi @Anmol-Baranwal, 

We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. 

Thank you for helping us maintain an authentic and beneficial space for everyone.

**Comment by darrylvdm6-prog** [comment]
https://github.com/orgs/community/discussions/44749#discussioncomment-9200190

**Comment by jidanni** [comment]
One might contemplate blocking specific users.
While suddenly one finally has peace from their constant notifications, one should be aware of
https://ux.stackexchange.com/questions/155316/how-to-improve-flawed-blocking-algorithm/155328
